File: TrabajoSep04/Java/Rectangulo.java
public class Rectangulo {
    double longitud;
    double ancho;
    double area;
    double perimetro;

    Rectangulo (double longitud, double ancho) {
        this.longitud = longitud;
        this.ancho = ancho;        
        this.area = this.calcularArea();
        this.perimetro = this.calcularPerimetro(); 
    }

    double calcularArea() {
        return this.ancho * this.longitud;
    }

    double calcularPerimetro() {
        return this.ancho * 2 + this.longitud * 2 ;
    }

    double cambiarAncho(double newAncho){
        return this.ancho = newAncho;
    }

    double cambiarLargo(double newLargo){
        return this.longitud = newLargo;
    }
}

File: TrabajoSep04/Cpp/Rectangulo.cpp
#include <iostream>

class Rectangulo {

public:
    double largo;
    double ancho;
    double area;
    double perimetro;

    Rectangulo (double newLargo, double newAncho) {
        largo = newLargo;
        ancho = newAncho;
        area = calcularArea();
        perimetro = calcularPerimetro();
    }

    double calcularArea(){
        return largo * ancho;
    }

    double calcularPerimetro() {
        return largo * 2 + ancho * 2;
    }

    void cambiarAncho(double newAncho){
        ancho = newAncho;
    }

    void cambiarLargo(double newLargo){
        largo= newLargo;
    }

};